models/tests/test_markov_chain.py

```diff
@@ -9,3 +13,42 @@ def test_generate_n_grams():
 
     assert model.n_grams_list == list(set([
         (1, 2, 3), (2, 3, 4), (3, 4, 5), (6, 7, 8), (7, 8, 9), (8, 9, 0)]))
+
+def test_data_loading():
+    model = models.MarkovChain()
```

Comment:
There are no `models` in this scope
```suggestion
    model = MarkovChain()
```